cmd/ui/src/ducks/explore/reducer.ts

```diff
@@ -50,36 +51,16 @@ const graphDataReducer = (state = initialGraphDataState, action: types.GraphActi
             draft.export = action.payload;
         } else if (action.type === types.TOGGLE_TIER_ZERO_NODE) {
             const systemTags = state.chartProps.items[action.nodeId].data.system_tags;
-            // remove the tier zero tag from the node
-            if (systemTags.includes('admin_tier_0')) {
-                // check if node is marked as owned or not
-                { systemTags.includes('owned') ?
-                    draft.chartProps.items[action.nodeId].data.system_tags = 'owned' :
-                    draft.chartProps.items[action.nodeId].data.system_tags = ''
-                }
-            } else {
-                // check if node is marked as owned or not
-                { systemTags.includes('owned') ?
-                    draft.chartProps.items[action.nodeId].data.system_tags = 'admin_tier_0 owned' :
-                    draft.chartProps.items[action.nodeId].data.system_tags = 'admin_tier_0'
-                }
-            }
+            if (systemTags.includes(TIER_ZERO_TAG)) {
```

Comment:
Instead of working with regex would it be preferred to use `splice` and do something like the following:
```typescript
let systemTags = []
// Check if system_tags contains tags then split, else leave empty
{ state.chartProps.items[action.nodeId].data.system_tags ?
	systemTags = state.chartProps.items[action.nodeId].data.system_tags.split(" ") : null }

if (systemTags.includes(TIER_ZERO_TAG)) {
	// Remove tag
	systemTags.splice(systemTags.indexOf(TIER_ZERO_TAG), 1);
	draft.chartProps.items[action.nodeId].data.system_tags = systemTags.join(' ');
} else {
	// Add tag
	systemTags.push(TIER_ZERO_TAG);
	draft.chartProps.items[action.nodeId].data.system_tags = systemTags.join(' ');
}
```

Reply:
Yea that looks nicer 👍